distributed/tests/test_actor.py

```diff
@@ -626,6 +635,27 @@ def prop(self):
         await ac.prop
 
 
+def test_as_completed(client):
+    class Simple:
+        state = 0
+
+        def inc(self):
+            self.state += 1
+            return self.state
+
+    ac = client.submit(Simple, actor=True).result()
+    futs = [ac.inc() for _ in range(10)]
```

Comment:
Nit: I prefer the pronoun `futures`.  This is what is commonly used throughout the codebase.  I'd prefer that we reduce the number of pronouns that we use if possible.

Reply:
and fut -> future below